is-logged-in.bs

```diff
@@ -18,18 +18,184 @@ Level: None
+
+At the same time, some user agents may wish to offer means to clear long-term storage where not the user is logged in, considering it a privacy issue that websites can store data virtually forever on the device. A logged-in signal could also enable a user agent to offer powerful browser features, or to create a seamless user experience by relaxing particular restrictions on websites where the user is logged in.
+
 </section>
 
 <h2 id="infra">Infrastructure</h2>
 
 This specification depends on the Infra standard. [[!INFRA]]
 
-<h2 id="acknowledgements" class="no-num">Acknowledgements</h2>
+<h2 id="the-isloggedin-api">The IsLoggedIn API</h2>
+
+The IsLoggedIn API creates a clear signal to the user agent that the user is logged into a website. It does <em>not</em> create a mechanism for the web developer to manage a user's identity or log the user in or out.
+
+This specification defines:
+
+  * A method for a website to signal to the browser that the user is logged in ({{Navigator/setLoggedIn()}})
+  * A method to signal that the user is logged out ({{Navigator/setLoggedOut()}})
+  * A method for third-party iframes to check whether the user is a logged-in customer ({{Navigator/setLoggedOut()}})
+
+ISSUE: Update "website" with a more precise term
+
+<h3 id="setLoggedIn">Logged-in signal: <dfn export method for=Navigator><code>setLoggedIn()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.setLoggedIn(
+    username: non-whitespace string of limited length,
+    credentialTokenType: “httpStateToken” OR “legacyAuthCookie”,
+    optionalParams { }
+) –> Promise<void>
+```
+
+The returned promise would resolve if the status was set and reject if
+not. The API could potentially take an expiry parameter but here we’re
+assuming that a designated [HTTP State
+Token](https://mikewest.github.io/http-state-tokens/draft-west-http-state-tokens.html)
+or “legacy auth cookie” manages the expiry of the login through their
+own mechanisms.
+
+<h3 id="setLoggedOut">Logged-out signal: <dfn export method for=Navigator><code>setLoggedOut()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.setLoggedOut(optionalUsername) –> Promise<void>
+```
+
+The optional username parameter highlights that we might want to support
+concurrent logins on the same website which would require the site to
+keep track of who to log out and credential tokens to be scoped to user
+names.
+
+<h3 id="isLoggedIn">Checking for a logged-in signal: <dfn export method for=Navigator><code>isLoggedIn()</code></dfn></h3>
+
+ISSUE: Flesh out section with explanatory text, an example, more formal/algorithmic language.
+
+```
+Navigator.isLoggedIn() –> Promise<bool>
+```
+
+<h2 id="defending-Against-abuse">Defending Against Abuse</h2>
```

Comment:
Did you mean to capitalize "Against" in the id?

Reply:
Nope! Thanks John.